### DISCUSSION TITLE
How to install and use RFID devices with Snipe-IT

### DISCUSSION THREAD

**Discussion by brett2403** [snipe/snipe-it#13611] [General]
### Is your feature request related to a problem? Please describe.

I understand that Snipe-IT already works with NFC/RFID tags, depending on how you implement the data on those tags, but i cannot find any documentation in the installation documents that indicates how to go about installing RFID with Snipe-IT?
Or what RFID devices, readers and tags are compatible

### Describe the solution you'd like

ideally a how to step by step or you tube tutorial as how to gain the benefits of RFID with Snipe-IT. How to install and setup.

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
